packages/firestore/test/integration/api_internal/bundle.test.ts

```diff
@@ -242,6 +242,28 @@ apiDescribe('Bundles', (persistence: boolean) => {
     });
   });
 
+  it('loaded documents should not be GC-ed right away', () => {
+    return withTestDb(persistence, async db => {
+      const builder = bundleWithTestDocsAndQueries(db);
+
+      const fulfillProgress: firestore.LoadBundleTaskProgress = await db.loadBundle(
+        builder.build('test-bundle', { seconds: 1001, nanos: 9999 })
+      );
+
+      verifySuccessProgress(fulfillProgress!);
+
+      // Read for a first time, expecting document in cache.
+      let snap = await db.collection('coll-1').get({ source: 'cache' });
+      verifySnapEqualTestDocs(snap);
```

Comment:
Reads from cache don't trigger GC (since they don't allocate targets). You would need to issue a read from the server here.

Reply:
Done.